http/src/client/builder.rs

```diff
@@ -101,7 +114,20 @@ impl ClientBuilder {
     ///
     /// The default is `false`.
     pub fn skip_ratelimiter(&mut self, skip_ratelimiter: bool) -> &mut Self {
-        self.skip_ratelimiter = skip_ratelimiter;
+        self.ratelimiter = if skip_ratelimiter {
+            RatelimiterBuilder::Skip
+        } else {
+            RatelimiterBuilder::New
+        };
+
+        self
+    }
+
+    /// Set whether to skip the client's ratelimiter before making the request.
+    ///
+    /// The default is `false`.
+    pub fn ratelimiter(&mut self, ratelimiter: Ratelimiter) -> &mut Self {
```

Comment:
I'd rather get rid of skip_ratelimiter and allow passing in an `Into<Option<Ratelimiter>>`, which would be the equivalent of combining skip_ratelimiter and ratelimiter

Reply:
Yep ok, I just figured this way there would not be any breaking API change.